README.md

```diff
@@ -118,7 +118,7 @@ rspec
 
 ## 🔭 Video presentations <a name="video-presentation"></a>
 
-- [Link](https://www.loom.com/share/c36ddcd15994058b73071bc912aaf9?sid=f1595b5-3028-4f78-8982-a091887e0c8)
+- [Link](https://drive.google.com/file/d/1aPMvTrDzDPvfbHZe3dYYs4XoJdMdcTka/view?usp=sharing)
```

Comment:
[REQUIRED] I am still having the problem previous reviewers had with your video presentation. It says I don't have access. Kindly follow this [link](https://support.google.com/drive/answer/2494822?hl=en&co=GENIE.Platform%3DDesktop) to learn more about sharing files.

![image](https://github.com/2004-okari/CATALOG/assets/62938735/7c083943-6b63-4bd9-bd90-d137250ca7ff)